scripts/systems.py

```diff
@@ -198,8 +198,8 @@
 """,
 }
 
-cscs["santis"] = {
-    "Cores": 288,
+cscs["alps-gh200"] = {
+    "Cores": 256,
```

Comment:
Keep this:
```suggestion
    "Cores": 288,
```
?

We can set `PIKA_THREADS=64` int the env vars?